src/Symfony/Bundle/SecurityBundle/Command/UserPasswordEncoderCommand.php

```diff
@@ -32,10 +35,10 @@ protected function configure()
     {
         $this
             ->setName('security:encode-password')
-            ->setDescription('Encode a password.')
+            ->setDescription('Encodes a password.')
             ->addArgument('password', InputArgument::OPTIONAL, 'Enter a password')
-            ->addArgument('user-class', InputArgument::OPTIONAL, 'Enter the user class configured to find the encoder you need.')
```

Comment:
And this description could be: `The User entity class path associated with the encoder used to encode the password`